docs/aerodromes/classd/Archerfield.md

```diff
@@ -69,6 +69,64 @@ The circuit direction changes depending on time of day and runway being used.
 
 Circuits are to be flown at `A010`.
 
+## Helicopter Operations
+Archerfield has three helipads located around the aerodrome:
+
+- **Central Helipad**, on the southern side of Taxiway Foxtrot
+- **Northern Helipad**, adjacent the north-eastern end of Runway 04R/22L
+- The **Queensland Government Air facility** used by rescue helicopters, north of Taxiway B5
+
+!!! note
+    Both the **Northern** and **Southern** helipads are inside the maneuvering area and treated like a runway, requiring a takeoff and landing clearance from AF ADC. Helicopters operating on other surfaces outside the maneuvering areas should be instructed to *"report airborne"*  or *"report on the ground"* so the controllers knows when they are active / no longer active in their airspace. 
+
+### Training Areas
+There are 3 designated helicopter training areas at Archerfield:
+
+**Area A**: North of Taxiway B within the confines of the aerodrome boundary, clear of all buildings, not above `A005`. Circuit direction and frequency as per ATIS. A maximum of 3 helicopters are permitted to operate circuits concurrently. Additionally, a maximum of 2 helicopters are permitted to operate in ground effect only.
+
+**Area B**: North and west of Runway 04L within the confines of the aerodrome boundary, clear of all buildings, not above `A005`. Circuit direction and frequency as per ATIS. A maximum of 3 helicopters are permitted to operate circuits concurrently. Additionally, a maximum of 2 helicopters are permitted to operate in ground effect only.
+
+**Area C**: South and east of Taxiway D, inside the fixed wing circuit for runway 04R/22L, east of the control tower, not above `A008`. Landings must be conducted north of Taxiway B. Once established, the only radio communications are an airborne call before each circuit and a downwind call by pilots. A maximum of 2 helicoptesr are permitted to operate in this area concurrently.
```

Comment:
Seems as though circuits are handled differently depending on the training area involved, do you have any guidance for controllers on how to handle this?

Reply:
I haven't been able to find any documentation or examples of phraseology on how it works. I was under the assumption that most of time for areas such as Alpha or Bravo where their isn't a requirement for calls, that ADC would just provide traffic statements while helicopters

Reply:
Not personally sure about that but will do some more research and see what i can find about it.

Reply:
I think this is the last thing we need to resolve and provide guidance on, and then we're pretty much set.